searx/templates/simple/result_templates/images.html

```diff
@@ -1,10 +1,8 @@
 <article class="result result-images {% if result['category'] %}category-{{ result['category'] }}{% endif %}">{{- "" -}}
         <a {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} href="{{ result.img_src }}">{{- "" -}}
-                <div class="image_thumbnail_container">
-                        <img class="image_thumbnail" {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} src="{% if result.thumbnail_src %}{{ image_proxify(result.thumbnail_src) }}{% else %}{{ image_proxify(result.img_src) }}{% endif %}" alt="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{- "" -}}
-                        {%- if result.resolution %} <span class="image_resolution">{{ result.resolution }}</span> {%- endif -%}
-                </div>
```

Comment:
> I'm not really sure what's causing this here 

I think this div (container) is the problem .. this div is not needed: when we have a span tag with the resolution directly after the img tag:

```html
<img class="image_thumbnail..> ..</img>
<span class="image_resolution">100x200</span>
```

the absolute position of the span tag is relative to this img tag:

```css
.image_resolution {
    position: absolute;
    right: 0;
}
```

I'm not a CSS expert, but it seems to work as expected .. can you please double check my WIP commit / thanks :+1:

Reply:
Yes, thanks, this does work as intended :+1: